certification/pyxis/types.go

```diff
@@ -18,6 +19,16 @@ type CertImage struct {
 	UncompressedTopLayerId string       `json:"uncompressed_top_layer_id,omitempty"` // TODO: figure out how to populate this, it is not required
 }
 
+type Links struct {
+	RpmManifest     Href `json:"rpm_manifest,omitempty"`
+	TestResults     Href `json:"test_results"`
```

Comment:
Shouldn't these all be omitempty? We would not want to submit these ever.

Reply:
same as above.

Reply:
no longer going to use these structs.